src/Field_One.php

```diff
@@ -132,9 +132,21 @@ public function ref($defaults = [])
         $m = $this->getModel($defaults);
         if ($this->owner->loaded()) {
             if ($this->their_field) {
-                return $m->loadBy($this->their_field, $this->owner[$this->our_field]);
+                return $m->tryLoadBy($this->their_field, $this->owner[$this->our_field])
+                    ->addHook('afterSave', function ($m) {
```

Comment:
Do we need afterSave hook here even if referenced model can be loaded? This is good if we allow to change `id` of model, but ... do we allow that?